src/routes/transactions/transactions.controller.spec.ts

```diff
@@ -250,14 +251,28 @@ describe('Transactions Controller (Unit)', () => {
       const chainId = faker.random.numeric();
       const safeAddress = faker.finance.ethereumAddress();
       const chainResponse = chainBuilder().with('chainId', chainId).build();
+      const safeAppsResponse = [
+        safeAppBuilder()
+          .with('url', 'https://apps.gnosis-safe.io/safe-claiming-app')
+          .with(
+            'iconUrl',
+            'https://safe-apps.dev.5afe.dev/safe-claiming-app/logo.svg',
+          )
+          .with('name', '$SAFE Claiming App')
+          .build(),
+      ];
```

Comment:
Did you consider using fake data here?

I don't really see the benefit right now of reading JSON files in the tests when we can compare the payloads with a TS/JS object here. Generated fake data would also provide different inputs on each run which can be valuable.

Reply:
I've rewritten this test using builders instead of JSON files in [d9abbce](https://github.com/5afe/safe-client-gateway-nest/pull/236/commits/d9abbce9dd91fd30e603198f8ea696748569eb1a). For the expected response I've just written the expected object in the test. An alternative approach would be using [Jest matchers](https://jestjs.io/docs/expect#expectobjectcontainingobject) to assert the parts of the response in which we are interested are correct. So, to summarise, I see several options to assert the response objects:
- Keep using JSON files for them.
- Use plain JS objects in each test.
- Use Jest matchers to check the interesting parts of the response.
- Use builders also for responses (i.e. use builder for route entities).

What do you think? 🙂

Reply:
I would say it's fine to have the play JS object as the expected object in each test.

I would try to avoid using internal tools (builders or jest) to represent the expected contract at this stage. We can reconsider this if we don't see value in having the complete payload in the expected response.

Reply:
Perfect, I agree we can use just plain JS objects for now. I think in other cases as I commented out [here](https://github.com/5afe/safe-client-gateway-nest/pull/215#issuecomment-1401752850) maybe it's better to use some matchers because the focus of the tests is not the mapping. But for this PR I think plain JS objects are the way to go since we're interested in the explicit mappings of transactions 👍